packages/nuxt/src/components/plugins/tree-shake.ts

```
@@ -15,12 +15,192 @@ interface TreeShakeTemplatePluginOptions {
+        }
+      }
+    }
+    return false
+  }
+
+  /**
+   * detect if the component is called else where
+   * ImportDeclarations and VariableDeclarations are ignored
+   * return the name of the component if is not called
+   */
+  function isComponentNotCalledInSetup (codeAst: Node, name: string): string | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)